src/app/public/modules/repeater/repeater-item.component.html

```diff
@@ -30,6 +30,25 @@
   #skyRepeaterItemLeft
 >
   <div class="sky-repeater-item-left">
+    <ng-container *ngIf="draggable">
+      <span
+        aria-live="assertive"
```

Comment:
It looks like `assertive` will read these instructions as they change, but will `assertive` also read the first time the page loads?

Reply:
The instructions for the beginning aren't tied to this section for that reason. This `span` is only used for once the spacebar is hit. Prior to that we use the description on the grab handle. The live region ensures that the new instructions are read out even though the information isn't on the grab handle element.

Reply:
Cool. Sounds like you've got that covered. Nice work.